example/fragment/gen/client.go

```diff
@@ -0,0 +1,56 @@
+// Code generated by github.com/Yamashou/gqlgenc, DO NOT EDIT.
+
+package gen
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/Yamashou/gqlgenc/clientv2"
+)
+
+type Client struct {
+	Client *clientv2.Client
+}
+
+func NewClient(cli *http.Client, baseURL string, interceptors ...clientv2.RequestInterceptor) *Client {
+	return &Client{Client: clientv2.NewClient(cli, baseURL, interceptors...)}
+}
+
+type Query struct {
+	Conversation *Conversation "json:\"conversation\" graphql:\"conversation\""
+}
+type Mutation struct {
+	Something string "json:\"something\" graphql:\"something\""
+}
+type ConversationStructType struct {
+	Hello *HelloWorldStructType "json:\"hello\" graphql:\"hello\""
+}
+type HelloWorldStructType struct {
+	HelloWorldMessage *string "json:\"helloWorldMessage\" graphql:\"helloWorldMessage\""
+}
+type ConversationByIDQuery struct {
+	Conversation *ConversationStructType "json:\"conversation\" graphql:\"conversation\""
```

Comment:
@adam-zethraeus this gives us an idea of what kind of code changes are required to generate named structs instead of anonymous ones. Not dealt with here are things like unions, interfaces and fragments.